### DISCUSSION TITLE
Two separate led strips from one controller

### DISCUSSION THREAD

**Discussion by robotm-team** [arendst/Tasmota#15639] [Q&A] [answered]
Hi.
I have an Arilux LC01 (GRB).
I want to connect 2 separate led strips to it.
One is CCT and the other is green.
Is it possible to set the controller in such a way that one of the tapes (CCT) works on 2 outputs with warm<>cold control + dimmer (TOGGLE1) and the other (in my case, green) only on dimmer (TOGGLE2)?
I would like it to look like the attached picture.
Thanks!
![tasmota question](https://user-images.githubusercontent.com/76626315/168577567-87982e8a-ea27-4da8-b511-7728e1dc64d8.jpg)

**Comment by s-hadinger** [comment, marked as answer]
No it's not supported per se. But you can define the green strip as PMW1, assign PWM2/3 to unused GPIO and PWM4/5 to CT. If you set `SetOption137 128`, the you can control separately RGB (green only matters in your case), and CT.

For the green channel, just use a full green Hue `Hue 120` and use the Dimmer channel to modulate brightness

**Reply by robotm-team** [reply]
Are you sure it's "SetOption137 128"? Maybe it was ""SetOption37 128""

**Reply by s-hadinger** [reply]
Correct `SetOption37 128`
